src/Views/Appearance.vala

```diff
@@ -193,15 +186,23 @@ public class PantheonShell.Appearance : Gtk.Grid {
             }
         }
 
+        var grid = new Gtk.Grid () {
+             column_spacing = 7, // Off by one with Gtk.RadioButton
+             row_spacing = 6,
```

Comment:
Extraneous leading spaces

Reply:
Oops copy/paste issue for sure. Thanks for catching